Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -210,6 +214,25 @@ public partial class AudioLink : MonoBehaviour
         private int _Samples3R;
         // ReSharper restore InconsistentNaming
 
+#if UNITY_WEBGL
```

Comment:
This file is already pretty crowded. Consider moving the WebGL specific stuff to a new file, like we do for the DataAPI already - AudioLink is a partial class

Reply:
do we have to?

Reply:
not blocking, but i think it would be nicer. Consider for a follow up

Reply:
sure